src/main/java/mate/Main.java

```diff
@@ -0,0 +1,9 @@
+package mate;
+
+public class Main {
+    public static void main(String[] args) {
+        for (int i = 0; i < 3; i++) {
+            System.out.println((i + 1) + " " + new Lottery().getRandomBall().toString());
```

Comment:
move lottery creation outside of the loop
```suggestion
        for (int i = 0; i < 3; i++) {
            System.out.println((i + 1) + " " + new Lottery().getRandomBall().toString());
```

Reply:
but in "Common mistakes" wrote, "Use for loop for creating several objects of the same class". That's why I did so. 

https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class

Reply:
Shoud I change the solution?